build.gradle

```diff
@@ -0,0 +1,326 @@
+        selection.reject('Release candidate')
+      }
+    }
+  }
+}
+
+// https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
+dependencyCheck {
+  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
+  // range of 0-10 fails the build, anything greater and it doesn't fail the build
+  failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 0 : 11
+  suppressionFile = 'config/owasp/suppressions.xml'
+}
+
+dependencyManagement {
+  dependencies {
+    dependency group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: '1.64'
+    // CVE-2018-10237 - Unbounded memory allocation
+    dependencySet(group: 'com.google.guava', version: '28.2-jre') {
+      entry 'guava'
+    }
+  }
+}
+
+repositories {
+  mavenLocal()
+  jcenter()
+  mavenCentral()
+  maven {
+    url "https://dl.bintray.com/hmcts/hmcts-maven"
+  }
+}
+
+dependencies {
+
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-json'
+ // compile group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: versions.springBoot
+  compile group: 'org.springframework.retry', name: 'spring-retry', version: '1.2.1.RELEASE'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: versions.springBoot
+  compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.5'
+
+  compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
+
+  compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'logging-httpcomponents', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.0.4'
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.2.RELEASE'
+
+  //compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
+  //compile(group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.4')
+  compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
+  compile 'org.projectlombok:lombok:1.18.2'
+
+  compile group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '2.5.6'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.0.pr3'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.0.pr3'
+  compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.0.pr2'
```

Comment:
can bump jackson to the latest

Reply:
spring boot compatible version is 2.10.2